.cloudbuild/scripts/cmd/nightly-tests/nightly.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import "log"
+
+func main() {
+	if err := run(); err != nil {
+		log.Fatalf("FAILED: %s", err.Error())
```

Comment:
nit:

```suggestion
		log.Fatalf("FAILED: %v", err)
```

Reply:
This was an artefact of an overzealous `git add` - thanks for spotting it.